include/seqan/basic/iterator_interface.h

```diff
@@ -350,43 +350,12 @@ position(TIterator const & it,
 
 template <typename T, typename TContainer>
 inline SEQAN_HOST_DEVICE bool
-atBegin(T const & it, TContainer const & cont)
-{
-    SEQAN_CHECKPOINT;
-    return it == begin(cont, Standard());
-}
-
-template <typename T, typename TContainer>
-inline SEQAN_HOST_DEVICE bool
-atBegin(T const & it, TContainer & cont)
-{
-    SEQAN_CHECKPOINT;
-    return it == begin(cont, Standard());
-}
-
-template <typename T, typename TContainer>
-inline SEQAN_HOST_DEVICE bool
 atBegin(T & it, TContainer & cont)
```

Comment:
Why keeping the non-const version instead of the const-version?
atBegin does neither change the iterator nor the container and returns a bool. Hence I would rather keep the very first one?